_emails/philadelphia.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Philadelphia, PA
+permalink: "/philadelphia"
+name: Letter to Mayor and Council Members
+state: PA
+city: Philadelphia
+layout: email
+recipients:
+- Cindy.Bass@Phila.gov
+- Kendra.Brooks@phila.gov
+- Kenyatta.Johnson@Phila.gov
+- Helen.Gym@phila.gov
+- Jamie.Gauthier@phila.gov
+- Maria.Q.Sanchez@phila.gov
+- Derek.Green@phila.gov
+- Katherine.Gilmore.Richardson@phila.gov
+- Allan.Domb@Phila.gov
+- Cherelle.Parker@Phila.gov
+- Isaiah.Thomas@phila.gov
+- Darrell.Clarke@phila.gov
+- David.Oh@Phila.gov
+- Brian.Oneill@phila.gov
+- Mark.squilla@phila.gov
+- brian.abernathy@phila.gov
+- james.kenney@phila.gov
+- bobby@bobbyhenon.com
```

Comment:
bobbyhenon.com is suspended and likely not receiving emails. Can be updated to phila.gov email -- bobby.henon@phila.gov